cmd/otelcorecol/builder-config.yaml

```
@@ -48,6 +49,7 @@ replaces:
   - go.opentelemetry.io/collector/extension/auth => ../../extension/auth
   - go.opentelemetry.io/collector/extension/ballastextension => ../../extension/ballastextension
   - go.opentelemetry.io/collector/extension/zpagesextension => ../../extension/zpagesextension
+  - go.opentelemetry.io/collector/extension/memorylimiterextension => ../../extension/memorylimiterextension
```

to preserve alphabetical order
```suggestion
  - go.opentelemetry.io/collector/extension/memorylimiterextension => ../../extension/memorylimiterextension
  - go.opentelemetry.io/collector/extension/zpagesextension => ../../extension/zpagesextension
```

---

reordered